nxviz/plots.py

```diff
@@ -286,6 +288,17 @@ class CircosPlot(BasePlot):
     """
     Plotting object for CircosPlot.
     """
+    def __init__(self, graph, **kwargs):
+        """Create the CircosPlot.
+
+        Accepts the following additional arguments apart from the ones in
+        `BasePlot`:
+
+        :param rotate_labels: Whether to rotate node labels.
+        :type node_color: `bool`
+        """
+        self.rotate = kwargs.pop("rotate_labels", False)
```

Comment:
I have usually seen lines like this come after the `super(...)...` line. Was there a reason for this being put above the line below?

Reply:
Just in case that `BasePlot` validates its arguments and would not accept he `rotate_labels` argument. But I think that is not the case so before or after the super should both work :)